tests/autologging/test_autologging_safety_unit.py

```diff
@@ -1168,12 +1172,22 @@ def test_validate_args_throws_when_extra_args_are_not_functions_classes_or_lists
 
     with pytest.raises(Exception, match="Invalid new input"):
         _validate_args(
-            user_call_args, user_call_kwargs, invalid_type_autologging_call_args, user_call_kwargs
+            "",
+            "",
```

Comment:
@MarkYHZhang Can we apply https://github.com/mlflow/mlflow/pull/5683/files#diff-c18d334f6497b607d53017ccc716f80d1f2ff2cd989358c5bca64ae877f545ccR1146-R1148 to the rest of this file as well? I.e. instead of passing empty strings, include content that helps maintainers interpret the meaning of these args.